pynest/nest/tests/test_NodeCollection.py

```diff
@@ -69,6 +69,23 @@ def test_list_to_NodeCollection(self):
         nc = nest.NodeCollection(node_ids_in)
         self.assertEqual(nc.tolist(), [2, 3, 5, 7, 8])
 
+    def test_NodeCollection_to_numpy(self):
+        """ Conversion from NodeCollection to numpy array """
+        if HAVE_NUMPY:
+            n_neurons = 10
+            nc = nest.Create('iaf_psc_alpha', n_neurons)
+
+            # direct array conversion
+            n_arr = np.array(nc)
+
+            self.assertTrue(np.array_equal(n_arr, nc.tolist()))
```

Comment:
To avoid the unhelpful `False is not True` message in the event of failure, either convert the NumPy array to a list and use `self.assertEqual()`, or use [`numpy.testing.assert_equal()`](https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_equal.html#numpy-testing-assert-equal).